src/key/key.js

```diff
@@ -257,8 +257,13 @@ class Key {
    * @async
    */
   async getSigningKey(keyID = null, date = new Date(), userID = {}, config = defaultConfig) {
-    await this.verifyPrimaryKey(date, userID, config);
     const primaryKey = this.keyPacket;
+    try {
+      await this.verifyPrimaryKey(date, userID, config);
+      helper.checkKeyRequirements(primaryKey, config);
+    } catch (err) {
+      throw util.wrapError('Error verifying primary key', err);
```

Comment:
```suggestion
      throw util.wrapError('Could not verify primary key', err);
```
(since this error will most likely be wrapped again, it sounds a bit stilted otherwise)